frontend/src/app/components/blocks-list/blocks-list.component.html

```diff
@@ -64,6 +66,9 @@ <h1 *ngIf="!widget" class="float-left" i18n="master-page.blocks">Blocks</h1>
               <span class="skeleton-loader" style="max-width: 60px"></span>
             </ng-template>
           </td>
+          <td *ngIf="auditAvailable" class="text-right" [ngClass]="{'widget': widget, 'legacy': !indexingAvailable}">
+            <app-amount [satoshis]="block.extras.expectedFees" [noFiat]="true" digitsInfo="1.2-2"></app-amount>
```

Comment:
I wonder whether it would be better to show the percentage change here? I find myself comparing the two columns a lot in this view.

Reply:
yes, I agree